nevergrad/instrumentation/multivariables.py

```diff
@@ -188,6 +188,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__()
         self.keywords: Tuple[Optional[str], ...] = ()
         self.probably_noisy = False  # True if for some reason we conjecture that the objective function is noisy.
```

Comment:
"probably" noisy is not that nice, you want it to be considered as noisy, or not.